src/libstd/sys/unix/ext/fs.rs

```diff
@@ -262,7 +263,8 @@ pub trait PermissionsExt {
     ///
     ///     permissions.set_mode(0o644); // Read/write for owner and read for others.
     ///     assert_eq!(permissions.mode(), 0o644);
-    ///     Ok(()) }
+    ///     Ok(()) 
```

Comment:
Please remove the trailing whitespace:
```suggestion
    ///     Ok(())
```

Reply:
updated